src/luxonis_ml/scheme/v1/config_building_blocks/base_models/metadata.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
+from .custom_base_model import CustomBaseModel
+from ..enums import *
+
+class Metadata(BaseModel):
+    name: str
+    backbone: str
+    platform: Platform
+    version: float
```

Comment:
I think we should make this version clear to be "config_version", which is what I assume it refers to?
In case we change the schema in the future.

Reply:
The version here would refer to the version of the schema. So in case we change how we define this config in the future, we can easily modify it while keeping the backwards compatibility. The depthai would know how to decode the config based on this version.